src/studio/src/designer/backend/Languages/ini/nb.ini

```
@@ -271,6 +273,7 @@ helper_text_for_input = Inputfelt brukes når brukeren skal føre inn tekst elle
 helper_text_for_image = Bildekomponenten er en komponent som presenterer bilder i applikasjonen din. Kilden til bildet kan enten være en absolutt URL eller en relativ URL dersom du ønsker å hoste bildet selv. Les mer om hvordan du gjør det i dokumentasjonen. Om du skal endre størrelse kan du bruke andre gyldige CSS enheter, ikke bare «%». Eksempel: 150px.
 helper_text_for_radio_button = Bruk radioknapper når du vil at brukeren kun skal kunne svare på ett av alternativene. Om brukeren skal kunne svare på flere må du bruke avkrysningsbokser. Er det veldig mange alternativer kan det lønne seg og bruke en nedtrekksmeny.
 helper_text_for_select = Dette er et komponent som ligger i Altinns designsystem. Fungerer som en avkrysningsboks.
+helper_text_for_nav_bar = Navigasjonsbar gil enkel tilgang til alle sider i en applikasjon. Alle sider i appen blir listet opp, så denne komponenten egner seg derfor ikke til bruk sammen med sporvalg.
```

- [ ] Fix small typo

```suggestion
helper_text_for_nav_bar = Navigasjonsbar gir enkel tilgang til alle sider i en applikasjon. Alle sider i appen blir listet opp, så denne komponenten egner seg derfor ikke til bruk sammen med sporvalg.
```